apps/whale-api/src/module.api/stats.controller.ts

```diff
@@ -82,13 +82,14 @@ export class StatsController {
       feeburn: burnInfo.feeburn.toNumber(),
       emissionburn: burnInfo.emissionburn.toNumber(),
       auctionburn: burnInfo.auctionburn.toNumber(),
-      paybackburn: burnInfo.paybackburn.toNumber(),
+      paybackburn: burnInfo.paybackburn,
```

Comment:
This need to be remapped into `Number` to prevent downstream regression.

@canonbrother can you look into which array within the string[] should be used as the defacto number.

Reply:
ohh okay.. means we gonna sum it up!!